docs/dev_guide/reference/command-line-options-ref/deprecated-options.rst

```diff
@@ -0,0 +1,30 @@
+.. _dep_opt:
+
+Deprecated Options
+==================
+
+Occasionally, tool options are marked as "deprecated." Deprecated options are
+still supported in the current release, but they may be unsupported in future releases.
+
+The following table lists |tool_name| command line options that are deprecated.
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Deprecated Option
+     - Suggested Replacement
+   * - .. include:: /_include_files/options_def.rst
+          :start-after: opt-custom-helper-name:
+          :end-before: desc-custom-helper-name:
+     - No replacement
+   * - .. include:: /_include_files/options_def.rst
+          :start-after: opt-no-cl-namespace-inline:

```

Comment:
--no-cl-namespace-inline has been already removed in the latest code.

Reply:
Done.